examples/capacitor/yarn.lock

```diff
@@ -9799,6 +9814,14 @@ react-virtuoso@^2.2.1, react-virtuoso@^2.8.2:
     "@virtuoso.dev/react-urx" "^0.2.12"
     "@virtuoso.dev/urx" "^0.2.12"
 
+react-virtuoso@^2.3.1:
```

Comment:
Can we remove the example yarn bumps? Not like any harm, but it pollutes the PR?

Reply:
Removed via git checkout